### DISCUSSION TITLE
Broadcast Service API documentation

### DISCUSSION THREAD

**Discussion by hpcolus** [NightscoutFoundation/xDrip#2895] [General]
The webserver is well documented [see link](https://github.com/NightscoutFoundation/xDrip/blob/master/Documentation/technical/Local_Web_Services.md) but there is no information on [Broadcast Service API](https://xdrip.readthedocs.io/en/latest/use/interapp/) except that 'This service allows broadcast of xDrip+ values as an API and can be used by external apps like WatchDrip+
Is there any documentation about it?

**Comment by psonnera** [comment]
This is handled by @bigdigital maybe he can help

**Reply by bigdigital** [reply]
Please check this pr https://github.com/NightscoutFoundation/xDrip/pull/2004 it has a little more description about Broadcast Service API
